server/core/src/templates/bash-template

```diff
@@ -263,8 +263,8 @@ run() {
     java_opts="${JAVA_OPTS}"
   fi
 
-  if [[ "$DAFFODIL_DEBUG_CLASSPATH" != "" ]]; then
-    app_classpath="$app_classpath:$DAFFODIL_DEBUG_CLASSPATH"
+  if [[ "$daffodilDebugClasspath" != "" ]]; then
```

Comment:
We don't have any actual style guidelines for this sort of thing. (Probably we should.)

The precedent we do have is that the CLI looks for DAFFODIL_CLASSPATH, and it is a documented feature. So perhaps we should stick with that convention, and revert this change?

Reply:
So the change most dealt with the linting of TypeScript. In the `daffodilDebugger.ts` file when we export the ENV variable it will give a warning about `DAFFODIL_DEBUG_CLASSPATH` as TypeScript wants it be camel cased. However, if we don't care about this I can revert this change

Reply:
I am more comfortable sticking with DAFFODIL_DEBUG_CLASSPATH, particularly if a new lint tool would let us control/remove the warning for this particular style guideline. 

But this is in bash code, so I'm not sure why typescript cares about it or is even able to parse it.

Reply:
So it doesn't care about it in this particular file the only file where it complains is in the `src/daffodilDebugger.ts` file but I will revert that now